Files/Dialogs/DecompressArchiveDialog.xaml

```diff
@@ -0,0 +1,50 @@
+﻿<ContentDialog
+    x:Class="Files.Dialogs.DecompressArchiveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:helpers="using:Files.Helpers"
+    xmlns:local="using:Files.Dialogs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Title="Select a destination and extract files"
+    CornerRadius="{StaticResource OverlayCornerRadius}"
+    DefaultButton="Primary"
+    PrimaryButtonText="Extract"
+    RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+    SecondaryButtonText="Cancel"
+    Style="{StaticResource DefaultContentDialogStyle}"
+    mc:Ignorable="d">
+
+    <Grid
+        MinWidth="400"
+        Margin="0,16,0,0"
+        ColumnSpacing="6"
+        RowSpacing="3">
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
```

Comment:
Can you use a StackPanel instead of columns?

Reply:
Alignment with stackpanel is messed up here so I use grid

Reply:
Can you share a screenshot? This seems like an unnecessary workaround.